src/taskList.js

```diff
@@ -0,0 +1,56 @@
+import Task from './task.js';
+
+export default class TaskList {
+  constructor(tasks) {
+    this.tasks = tasks.map((el) => new Task(el.description, el.completed, el.index));
+  }
+
+  show = () => {
+    const toDoList = document.querySelector('.items');
+    toDoList.innerHTML = '';

```

Comment:
Try to use a do-while function instead of single quotes to clean the HTML. It will help your code to like more professional and improve the speed of this execution.

Reply:
The suggested approach was implemented.